src/test/scenarios/attestation/configuration/readme.cli.md

```diff
@@ -9,6 +9,7 @@ cli:
     - name: Operations_List
     - name: AttestationProviders_Create
     - name: AttestationProviders_Get
+    - function: mytest
```

Comment:
I think to support manual test means to put them in manual folder right?

Reply:
just like the commands split into manual folder and generated folder. it will try to load the generated folder first then load manual folder, so that manual command definition will overwrite the generated command

Reply:
Yes. All the steps in test-scenario can be overwrite by code in the manual folder. The difference between "name/function" items are: name refer to an example, function refer to an fully customized step.